components/Register/RegisterPage.js

```diff
@@ -0,0 +1,163 @@
+/* eslint react/jsx-indent-props:0 */
+/* eslint arrow-parens:0 */
+/* eslint react/no-unused-state:0 */
+/* eslint space-before-function-paren:0 */
+import axios from 'axios';
+import React, { useState } from 'react';
+import {
+    CssBaseline, Grid, Paper,
+    Typography, TextField, Button, Link
+} from '@material-ui/core';
+import useStyles from './RegisterStyle';
+
+const RegisterPage = () => {
+    const [userEmail, setEmail] = useState('');
+    const [userFirstName, setFirstName] = useState('');
+    const [userLastName, setLastName] = useState('');
+    const [userPassword, setPassword] = useState('');
+    const [userPhonerNumber, setphoneNumber] = useState('');
+    const [message, setMessage] = useState('');
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        if (userEmail !== '' && userFirstName !== '' && userLastName !== ''
+        && userPassword !== '' && userPhonerNumber !== '') {
+            const data = {
+                email: userEmail,
+                firstName: userFirstName,
+                lastName: userLastName,
+                password: userPassword,
+                phoneNumber: userPhonerNumber,
+            };
+            axios.post('/api/v1/user', data)
+                .then(() => {
+                    setMessage('Registration Successful. You can Login');
+                    setEmail('');
+                    setFirstName('');
+                    setLastName('');
+                    setPassword('');
+                    setphoneNumber('');
+                })
+                .catch((error) => {
+                    setMessage(error);
+                });
+        }
+    };
+
+    const classes = useStyles();
```

Comment:
do not use this pattern, we are not using in-line styles because they get recreated every time that this component is re-instantiated